pkg/odo/util/cmdutils.go

```diff
@@ -162,7 +162,24 @@ func PrintComponentInfo(client *occlient.Client, currentComponentName string, co
 		// Gather the output
 		var output string
 		for _, linkedService := range componentDesc.Status.LinkedServices {
-			output += fmt.Sprintf(" · %s\n", linkedService)
+
+			// Let's also get the secrets / environment variables that are being passed in.. (if there are any)
+			secrets, err := client.GetSecret(linkedService, project)
+			LogErrorAndExit(err, "")
+
+			if len(secrets.Data) > 0 {
+				// Iterate through the secrets to throw in a string
+				var secretOutput string
+				for i := range secrets.Data {
+					secretOutput += fmt.Sprintf("    · %v\n", i)
```

Comment:
should we use `\t` here instead of providing spaces?

Reply:
Nope, spaces will keep it consistent in output on Windows + Linux.

Reply:
Wouldn't using `\t` keep the output consistent and predictable? :thinking:

Reply:
Yeah, that was my thinking

Reply:
But we can merge it